### ISSUE TITLE
Allow `disable` without `enable` when disable covers the whole file

### ISSUE THREAD

**Issue by futpib** [mysticatea/eslint-plugin-eslint-comments#7]
Use case is that one may want to allow disabling a rule for the whole file but still catch accidentally not closed `eslint-disable` in the middle of a file.

With something like `"eslint-comments/disable-enable-pair":·["error", {"allowWholeFile": true}]`

This should be allowed:
```js
// @flow
/* eslint-disable no-unused-expressions */

import { Error } from '.';

const e = new Error();

e.name;
e.message;
e.stack;

e.id;

console.log(new Error('ooops'));

// $ExpectError
console.log(new Error(3));

// $ExpectError
e.topkek;
```

But this should be an error:
```js
// @flow

import { Error } from '.';

const e = new Error();

/* eslint-disable no-unused-expressions */
e.name;
e.message;
e.stack;

e.id;

console.log(new Error('ooops'));

// $ExpectError
console.log(new Error(3));

// $ExpectError
e.topkek;
```

**Comment by mysticatea** [comment]
Sounds good to me.
